src/inline.js

```diff
@@ -42,10 +43,13 @@ export default class InlineEditor extends StandardEditor {
 	 * @returns {Promise}
 	 */
 	destroy() {
-		this.updateEditorElement();
+		// Cache the data, then destroy. There's no way to tell whether model->view
```

Comment:
Cache the data, then destroy. It's safe to assume that the model->view conversion will not work after destroy().